styles.css

```diff
@@ -278,6 +276,10 @@ button:hover {
   padding: 15px;
 }
 
+.hide {
+  display: none;
+}
+
 .about-me {
   width: 100%;
   position: relative;
```

Comment:
- [ ] As mentioned before, consider to add background image to about section to match the design